src/portal-components/GenerateComponentsFromRow.tsx

```diff
@@ -227,6 +228,7 @@ export default class GenerateComponentsFromRow extends React.Component<
   private renderSynapseObjectFromData(el: RowSynapseConfig): React.ReactNode {
     const { queryResultBundle, entityHeaders } = this.state
     const { columnName = '', resolveSynId, props } = el
+    const deepCloneOfProps = cloneDeep(props)
```

Comment:
Michael says another approach would be to return a new props object (instead of cloning here).